src/packages/@support/features/roadmap/layout/Roadmap.svelte

```diff
@@ -147,10 +157,10 @@
       <div class="filter">
         <Select
           data={[
-            { name: "All Categories", id: "allCategories" },
             { name: "Feature Request", id: FeedbackType.FEATURE_REQUEST },
             { name: "UI Improvement", id: FeedbackType.UI_IMPROVEMENT },
-            { name: "Bug", id: FeedbackType.BUG },
+            { name: "Bugs", id: FeedbackType.BUG },
+            { name: "All Categories", id: "allCategories" },
           ]}
           onclick={(id = "") => {
```

Comment:
take id from enum

Reply:
I have directly used here because in many places isha has assigned this to uncategorized and i have marked this as allCategories

Reply:
need to be rectified

Reply:
okay

Reply:
Resolved